docs/process/teams.md

```diff
@@ -0,0 +1,28 @@
+# Teams
+
+**Who do you `@` for what? See below!**
+
+_(These teams are not set in stone and will likely evolve as we continue to iterate on what makes the most sense for the success of the product.)_
+
+## General Teams
+
+These are good teams to start with for general communication and questions. (Members of these teams are also members of the skills-based teams below.)
+
+| Team | Purpose |
+|:--|:--|
+| `@desktop/maintainers` | The people designing, developing, and driving GitHub Desktop. Includes all groups below. |
+| `@desktop/comrades` | Community members with a track record of activity in the Desktop project |
+
+## Skills-based Teams
```

Comment:
Any qualms about changing this heading to _Special-purpose Teams_

Reply:
i like that better! @iAmWillShepherd